code/__DEFINES/machines.dm

```diff
@@ -98,8 +98,8 @@
 //Program categories
 #define PROGRAM_CATEGORY_CREW "Crew"
 #define PROGRAM_CATEGORY_ENGI "Engineering"
-#define PROGRAM_CATEGORY_ROBO "Robotics"
 #define PROGRAM_CATEGORY_SUPL "Supply"
+#define PROGRAM_CATEGORY_SCI  "Science"
 #define PROGRAM_CATEGORY_MISC "Other"
```

Comment:
????

Reply:
code: Created a new program tab "Science" for the downloader app. Removed Robotics.